src/Clients/user.py

```diff
@@ -6,6 +6,7 @@
 '''
 
 # External imports
+import numpy as np
```

Comment:
Adding a full new dependency for a single module?

Would this be better done using standard python methods and builtins?

Otherwise, the introduction of a root dependency, to improve error checking across the application?

Reply:
Do you have any other idea to check null? I know only about numpy for that purpose

Reply:
It is done from my side. Do let  me know if you still  want some changes.